aldryn_jobs/migrations/0012_auto_20150713_0859.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import cms.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0012_auto_20150607_2207'),
```

Comment:
@cyberstar yes I fixed this actually within the other migration before rebase: https://github.com/aldryn/aldryn-jobs/commit/76f1210f0f0b6221ccf6dabae17a808719ac42be